perfkitbenchmarker/publisher.py

```diff
@@ -31,6 +31,7 @@
 from perfkitbenchmarker import flag_util
 from perfkitbenchmarker import version
 from perfkitbenchmarker import vm_util
+from elasticsearch import Elasticsearch
```

Comment:
This will cause an import error on machines without the elasticsearch module installed. Please move this import statement inside of `ElasticsearchPublisher` instead.